package.json

```
@@ -21,35 +31,30 @@
   ],
   "scripts": {
     "lint": "eslint .",
-    "prepare": "node ./npm-prepare.js",
+    "prepare": "node ./npm-prepare.cjs",
     "release:generate:latest": "eslint-generate-release",
     "release:generate:alpha": "eslint-generate-prerelease alpha",
     "release:generate:beta": "eslint-generate-prerelease beta",
     "release:generate:rc": "eslint-generate-prerelease rc",
     "release:publish": "eslint-publish-release",
-    "test": "nyc _mocha -- -c tests/{examples,lib}/**/*.js --timeout 30000"
+    "test": "c8 mocha tests/**/*.test.js --timeout 30000"
```

It seems that, when this is expanded by shell, the glob doesn't match files that are directly in the `tests` directory (`tests/plugin.test.js` and `tests/processor.test.js`), so only `tests/examples/all.test.js` are run on Ubuntu.

https://github.com/eslint/eslint-plugin-markdown/actions/runs/9980737397/job/27582824596

```
Run npm run test
  npm run test
  shell: /usr/bin/bash -e {0}

> eslint-plugin-markdown@5.1.0 test
> c8 mocha tests/**/*.test.js --timeout 30000



  examples
    react
      ✔ reports errors on code blocks in .md files (477ms)

  examples
    typescript
      ✔ reports errors on code blocks in .md files (1050ms)


  2 passing (2s)
```

(whereas on Windows, where there's no shell expansion, it's `139 passing (3s)`: https://github.com/eslint/eslint-plugin-markdown/actions/runs/9980737397/job/27582826878)

Maybe we could add quotes to always pass the glob itself to mocha:

```suggestion
    "test": "c8 mocha \"tests/**/*.test.js\" --timeout 30000"
```